diff_cover/__init__.py

```diff
@@ -1,5 +1,5 @@
-# This file is imported in setup.py and cannot use any external dependencies
+import pkg_resources
```

Comment:
With this line we have a dependency on setuptools. It is not listed in tool.poetry.dependencies

Reply:
Good catch. Looks like I can do it with `importlib`

Reply:
bah, only supported with python < 3.8 and I dont really wanna add a new dependency. 

Ill just specify setuptools, I would expect just about everyone who has diff_cover has pip which has setuptools

Reply:
turns out ill need pkg_resources for another couple of years. The importlib functionality did not get added till 3.8. 

There is a package that backports it but since anyone who has pip should have pkg_resources I went ahead and kept this solution. To keep deps technically correct I specified the earliest possible version of setuptools that still works. I am _fairly_ sure this should not break anyone unless their version of pip is ancient.

Reply:
I agree that this is a good option